conn_go18.go

```diff
@@ -5,32 +5,27 @@ package clickhouse
 import (
 	"context"
 	"database/sql/driver"
-	"io/ioutil"
-	"net/http"
-	"net/url"
-	"strings"
 )
 
 // Ping implements the driver.Pinger
 func (c *conn) Ping(ctx context.Context) error {
 	if c.transport == nil {
 		return driver.ErrBadConn
 	}
-	// make request with empty body, response should be "Ok"
-	u := &url.URL{Scheme: c.url.Scheme, User: c.url.User, Host: c.url.Host, Path: "/"}
-	req, err := http.NewRequest(http.MethodGet, u.String(), nil)
+
+	req, err := c.buildRequest(ctx, "select version()", nil, true)
```

Comment:
select 1 will not be enough ?

Reply:
I thought about it, may be yes, it'll be even better since we can scan a body of a response and check it to be exactly "1".